subsys/senss/Kconfig

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SENSS
+	bool "Sensor Subsystem"
+	default y
+	depends on DT_HAS_ZEPHYR_SENSS_ENABLED
+	select RING_BUFFER
+	help
+	  Enable Sensor Subsystem.
+
+if SENSS
+
+module = SENSS
+module-str = senss
+source "subsys/logging/Kconfig.template.log_config"
+
+config SENSS_MAX_SENSOR_COUNT
+	int "maximum sensor count for senss"
+	depends on SENSS
+	default 128
+	help
+	  This is the maximum sensor count supported for senss
+	  Typical values are 128
+
+config SENSS_MAX_REPORTER_COUNT
+	int "maximum reporters count for an client sensor"
+	depends on SENSS
+	default 10
+	help
+	  This is the maximum sensor reporeters count supported for a sensor
```

Comment:
Can you elaborate here, my understanding is that "reporter" means a sensor that we get data from. Though I believe that here too, we can get this number statically from devicetree.

Reply:
get reporter num from device tree now